Automattic/wp-calypso
# Features list: Fix text under /plans page

**Proposed Changes**

Change text under `/plans` inside `wp-admin` in order to reflect what exists
under `/pricing` page.

_**Important Notice**_

In order to show this feature, I needed to add it under `getPlanCompareFeatures` function for
both `Business` and `eCommerse` plans. (The above function is located on `plans-list.tsx` file, and exists on every plan).
However I don't really know if this affects anything, anywhere else, so this is **something that needs to be confirmed before** we merge this ticket.

In comparison, under `/start/plans` page, the  `getSignupCompareAvailableFeatures` is called
instead on desktop (and `getSignupFeatures` on mobile).

#### Testing Instructions

How to test:
- Visit [wordpress.com/plans/](http://wordpress.com/plans/)<your.site>.
- Note that feature text under Business and eCommerce plans is absent. (This step is about checking current behavior that is not desired)
- Pull branch, run Calypso on local
- Visit /plans/<your.site> locally.
- Note that feature with text SFTP, SSH, WP-CLI, and Database access under Business and eCommerse plans should have been added.

In order to provide as much help as possible here are some _before_ and _after_ screenshots:

`/plans` was _before_ the fix:

![before](https://user-images.githubusercontent.com/497103/194056571-4816b578-9712-4783-ae5a-2b80490ae717.png)

And _after_ the fix:

![after](https://user-images.githubusercontent.com/497103/194056557-5f0289fa-0cd4-4715-bbe1-c0a8ebfce6c3.png)

#### Pre-merge Checklist


Related to #68192
